Prox/Prox/Data/EventsController.swift

```diff
@@ -15,15 +15,15 @@ class EventsProvider {
     lazy var eventsDatabase: EventsDatabase = FirebaseEventsDatabase()
 
     private lazy var radius: Double = {
-        return RemoteConfigKeys.getDouble(forKey: RemoteConfigKeys.eventSearchRadiusInKm)
+        return RemoteConfigKeys.eventSearchRadiusInKm.value
     }()
 
     private lazy var eventStartNotificationInterval: TimeInterval = {
-        return RemoteConfigKeys.getTimeInterval(forKey: RemoteConfigKeys.eventStartNotificationInterval)
+        return RemoteConfigKeys.eventStartNotificationInterval.value * 60
```

Comment:
Maybe we should have `RemoteConfigTimeInterval` or `RemoteConfigMinutesToSeconds` that'd do this fragile multiplication for us?